documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// assembly-config-mirrormaker2.adoc
+
+[id='con-config-mirrormaker2-connectors-{context}']
+= Connector configuration
+
+[role="_abstract"]
+Use Mirrormaker 2.0 connector configuration for the internal connectors that orchestrate the synchronization of data between Kafka clusters.
+
+The following table describes connector properties and the connectors you configure to use them.
+
+.MirrorMaker 2.0 connector configuration properties
+[cols="4a,2,2,2",options="header"]
+|===
+
+|Property
+|sourceConnector
+|checkpointConnector
+|heartbeatConnector
+
+
+|admin.timeout.ms:: Timeout for admin tasks, such as detecting new topics. Default is `60000` (1 minute).
+|✓
+|✓
+|✓
+
+|consumer.poll.timeout.ms:: Timeout when polling the source cluster. Default is `1000` (1 second).
+|✓
+|✓
+|
+
+|offset-syncs.topic.location:: The location of the `offset-syncs` topic. Default is the _source_ cluster.
+|✓
+|✓
+|
+
+|replication.policy.class:: Policy to define the remote topic naming convention. Default is `org.apache.kafka.connect.mirror.DefaultReplicationPolicy`.
+|✓
+|✓
+|
+
+|replication.policy.separator:: The separator used for topic naming in the target cluster. Default is `.` (dot).
+|✓
+|✓
+|
```

Comment:
It's also used by for heartbeats. Should we also mention it's only used if `replication.policy.class` is `org.apache.kafka.connect.mirror.DefaultReplicationPolicy`

Reply:
Updated. Makes sense to add the additional note.